hack/make/test-docker-py

```diff
@@ -17,6 +17,7 @@ source hack/make/.integration-test-helpers
 --deselect=tests/integration/api_exec_test.py::ExecDemuxTest::test_exec_command_tty_stream_no_demux \
 --deselect=tests/integration/api_build_test.py::BuildTest::test_build_invalid_platform \
 --deselect=tests/integration/api_image_test.py::PullImageTest::test_build_invalid_platform \
+--deselect=tests/integration/api_image_test.py::PullImageTest::test_pull_invalid_platform \
```

Comment:
ah! thx!

Reply:
I think the one above may be incorrect then

Reply:
opened as a separate PR so that we get master to go green again while this is still being reviewed https://github.com/moby/moby/pull/39714